src/engraving/dom/engravingitem.cpp

```diff
@@ -1288,13 +1288,13 @@ bool EngravingItem::appliesToAllVoicesInInstrument() const
            && getProperty(Pid::VOICE_ASSIGNMENT).value<VoiceAssignment>() == VoiceAssignment::ALL_VOICE_IN_INSTRUMENT;
 }
 
-void EngravingItem::setInitialTrackAndVoiceAssignment(track_idx_t track)
+void EngravingItem::setInitialTrackAndVoiceAssignment(track_idx_t track, bool ctrlModifier)
```

Comment:
Wouldn't it be preferable to give this a name that describes the functionality instead of the UI that triggers the functionality? Like `currentVoiceOnly`?